fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -10,6 +10,8 @@ class CreateKeychainAction < Action
       def self.run(params)
         escaped_password = params[:password].shellescape
 
+        should_log = FastlaneCore::Globals.verbose?
```

Comment:
I have some concerns about logging the commands and responses to `security`, as they could wind up as part of logs on a third party service.

If the purpose of this logging is to aid in debugging, I think we may want to control this logging via a more specific control than `--verbose`. My fear is that people are already using `--verbose` and suddenly start logging sensitive information that they were not before.

Does it make sense to trigger this with a specific environment variable, perhaps?